src/FunctionLoader.cs

```diff
@@ -57,11 +58,23 @@ internal static void SetupWellKnownPaths(FunctionLoadRequest request)
             var appLevelModulesPath = Path.Join(FunctionAppRootPath, "Modules");
             var workerLevelModulesPath = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "Modules");
             FunctionModulePath = $"{appLevelModulesPath}{Path.PathSeparator}{workerLevelModulesPath}";
-
+            AddLatestAzModulesPath();
             // Resolve the FunctionApp profile path
             var options = new EnumerationOptions { MatchCasing = MatchCasing.CaseInsensitive };
             var profiles = Directory.EnumerateFiles(FunctionAppRootPath, "profile.ps1", options);
             FunctionAppProfilePath = profiles.FirstOrDefault();
         }
+
+        /// <summary>
+        /// Adds the latest AzModules path to FunctionModulePath
+        /// </summary>
+        private static void AddLatestAzModulesPath()
+        {
+            if (!string.IsNullOrWhiteSpace(FunctionModulePath)
+                && !string.IsNullOrWhiteSpace(latestAzModulePath))
+            {
+                FunctionModulePath = $"{FunctionModulePath}{Path.PathSeparator}{latestAzModulePath}";
```

Comment:
This can probably be moved up to the line that `FunctionModulePath` gets defined. You'll probably want a function later, but that can be added when this code becomes more complex.

Reply:
try to keep this modular, I can move up in the same line and refactor later on